.github/workflows/kfp-sdk-runtime-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: KFP Runtime Code Tests
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-sdk-runtime-tests.yml'
+      - 'sdk/python/**'
+      - 'test/presubmit-test-kfp-runtime-code.sh'
+
+jobs:
+  kfp-runtime-tests:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Run KFP Runtime Code Tests
+        run: |
+          export PULL_NUMBER="${{ github.event.inputs.pull_number || github.event.pull_request.number }}"
```

Comment:
Why do we need this?

Reply:
The runtime test .py code requires a PR number to be specified: https://github.com/kubeflow/pipelines/blob/master/sdk/runtime_tests/execute_commands_args_test.py#L125

If we don't specify the PR number, it fails. For instance, find this failed job on my fork: https://github.com/DharmitD/data-science-pipelines-argo/actions/runs/9861372467/job/27229672665

Having said that, I just realized I'm setting the var twice here, will remove the export statement.

Reply:
Done, removed the duplicate env var setting statement.